code/modules/clothing/spacesuits/miscellaneous.dm

```diff
@@ -552,3 +553,67 @@ Contains:
 
 /obj/item/clothing/suit/space/hardsuit/skinsuit/attackby(obj/item/I, mob/user, params)
 	return
+
+
+////IDPD HARDSUITS////
+////Grunt
+/obj/item/clothing/suit/space/hardsuit/ert/idpd
+	name = "grunt suit"
+	desc = "An iconic blue and silver hardsuit."
+	icon = 'icons/mob/IDPD.dmi'
+	icon_state = "idpd_grunt_suit"
+	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/ert/idpd
+	allowed = list(/obj/item/gun, /obj/item/ammo_box, /obj/item/ammo_casing, /obj/item/melee/baton, /obj/item/restraints/handcuffs, /obj/item/tank/internals)
+	armor = list("melee" = 50, "bullet" = 35, "laser" = 35, "energy" = 35, "bomb" = 35, "bio" = 100, "rad" = 100, "fire" = 65, "acid" = 65, "stamina" = 55)
+	slowdown = 0
+	strip_delay = 130
+	resistance_flags = FIRE_PROOF
+	jetpack = null
+
+/obj/item/clothing/head/helmet/space/hardsuit/ert/idpd
+	name = "grunt helmet"
+	desc = "a standard-issue I.D.P.D helmet"
+	icon = 'icons/mob/IDPD.dmi'
+	icon_state = "idpd_grunt_helmet"
+	armor = list("melee" = 50, "bullet" = 35, "laser" = 35, "energy" = 35, "bomb" = 35, "bio" = 100, "rad" = 100, "fire" = 65, "acid" = 65, "stamina" = 55)
+	strip_delay = 130
+	brightness_on = 7
+	resistance_flags = FIRE_PROOF
+
+////Chief
+/obj/item/clothing/suit/space/hardsuit/ert/idpd/chief
+	name = "chief suit"
+	desc = "Wearing this makes you feel #1"
+	icon_state = "idpd_chief_suit"
+	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/ert/idpd/chief
+	armor = list("melee" = 80, "bullet" = 70, "laser" = 70, "energy" = 70, "bomb" = 70, "bio" = 100, "rad" = 100, "fire" = 90, "acid" = 90, "stamina" = 90)
+
+/obj/item/clothing/head/helmet/space/hardsuit/ert/idpd/chief
+	name = "chief helmet"
+	desc = "Glorious"
+	icon_state = "idpd_chief_helmet"
```

Comment:
Helmets and hardsuits should probably have the same armor values

Reply:
I agree

Reply:
accident